data/categories.txt

```diff
@@ -9077,6 +9077,45 @@ sw:4Maji ya kunywa
 fa:اب اشامیدنی
 mr:पिण्याचे पाणी
 
+natural-hot_spring|@water
+en:3Hot Spring
+ar:ربيع حار
+be:4Гарачая крыніца|4Геатэрмальная крыніца
+bg:Горещ извор
+ca:Font termal|Font calenta|Deu termal
+cs:Termální pramen
+da:Varm kilde
+de:Heiße Quelle|Thermalquelle
+el:Ιαματικές πηγές
```

Comment:
after some more research I think it is better to have it like this:
```suggestion
el:Θερμή πηγή
```

Reply:
We can keep both variants for search (categories), but keep only one for description (type_strings.txt)

Reply:
then I would suggest to have `Θερμή πηγή` in `type_strings.txt` and `Θερμή πηγή|Ιαματική πηγή` in the search categories.